cmd/bridge/main.go

```diff
@@ -189,6 +223,12 @@ func main() {
 		CustomLogoFile:       *fCustomLogoFile,
 		StatuspageID:         *fStatuspageID,
 		DocumentationBaseURL: documentationBaseURL,
+		AlertmanagerURL:      alertmanagerURL,
+		GrafanaURL:           grafanaURL,
+		PrometheusURL:        prometheusURL,
```

Comment:
Sorry, I forgot that we already had a `prometheusBaseURL`. I would make each of these `alertmanagerPublicURL`, `prometheusPublicURL`, etc., to make the difference clear.